src/components/Charts/ModelChart.js

```diff
@@ -49,35 +50,11 @@ const ModelChart = ({
     },
     visible: !forCompareModels,
     condensedLegend: {
-      bgColor: projections.getChartSeriesColorMap().limitedActionSeries,
+      bgColor: COLORS.LIMITED_ACTION,
```

Comment:
These condensedLegend.bgColor values are used in the embed legend and weren't using the same constants that the charts now use (as of phase 3), so they could be wrong in some cases.